dask/templates/additional-worker-deployment.yaml

```diff
@@ -25,74 +25,81 @@ spec:
         app: {{ template "dask.name" $ }}
         release: {{ $.Release.Name | quote }}
         component: worker
-      {{- if $worker.annotations }}
+      {{- with $worker.annotations }}
       annotations:
-        {{- toYaml $worker.annotations | nindent 8 }}
+        {{- . | toYaml | nindent 8 }}
       {{- end }}
```

Comment:
Use of `with`.

The `with` is a conditional like if but will sets the context (`.`) to the truthy value passed when rendering its internals. It helps us avoid re-referencing the value we check for truthyness in. Doing this is common practice now days in Helm charts.

Reply:
Putting `toYaml` after the content to render.

```diff
-{{- toYaml $worker.annotations | nindent 8 }}
+{{- $worker.annotations | toYaml | nindent 8 }}
```

This improves readability in my mind, and feels a bit more consistent - otherwise one could also do `nindent` to the left without using a pipe symbol.